addons/l10n_be_hr_payroll/report/hr_individual_account_reports.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.tools import float_round
+
+
+class IndividualAccountReport(models.AbstractModel):
+    _name = 'report.l10n_be_hr_payroll.report_individual_account'
+    _description = 'Individual Account Report'
+
+    def _month_to_number(self, month):
+        month_dict = dict({
+            'January': 1,
+            'February': 2,
+            'March': 3,
+            'April': 4,
+            'May': 5,
+            'June': 6,
+            'July': 7,
+            'August': 8,
+            'September': 9,
+            'October': 10,
+            'November': 11,
+            'December': 12})
+        return month_dict[month.split()[0]]
+
+    def _get_data_from_report(self, data):
```

Comment:
Data is not coming from report actually. 

You could rename this into '_get_report_data'.